dev/tasks/conda-recipes/azure.osx.yml

```diff
@@ -40,6 +40,8 @@ jobs:
       export CI=azure
       source run_conda_forge_build_setup
       conda update --yes --quiet --override-channels -c conda-forge -c defaults --all
+      #XXX: pin conda
```

Comment:
Instead of "XXX", please point to JIRA issue and diagnostic of root cause.

Reply:
Will do, just waiting for the results :)